cmd/neofs-node/reputation.go

```diff
@@ -29,6 +29,7 @@ import (
 	truststorage "github.com/nspcc-dev/neofs-node/pkg/services/reputation/local/storage"
 	reputationrpc "github.com/nspcc-dev/neofs-node/pkg/services/reputation/rpc"
 	"github.com/nspcc-dev/neofs-node/pkg/util/logger"
+	apireputation "github.com/nspcc-dev/neofs-sdk-go/reputation"
```

Comment:
`reputationSDK`?

Reply:
In master `apireputation` is used as package alias instead of `reputationSDK`, but I don't mind.